test/system/doc_examples_axe_test.rb

```diff
@@ -3,20 +3,22 @@
 require "application_system_test_case"
 
 class IntegrationDocExamplesAxeTest < ApplicationSystemTestCase
-  # Skip components that should be tested as part of a larger component.
+  # Skip components that should be tested as part of a lartest/system/doc_examples_axe_test.rbger component.
   NOT_STANDALONE = [:AutoCompleteItemPreview].freeze
 
   # Starting point violations to be addressed. Do not add to this list!
   STARTING_POINT_VIOLATIONS = [:MarkdownPreview, :AutoCompletePreview, :TabComponentPreview, :NavigationTabComponentPreview].freeze
 
   def test_accessibility_of_doc_examples
+    # Workaround to ensure that all component previews are loaded.
+    visit("/rails/view_components")
+
+    raise "You must generate previews with `bundle exec rake docs:preview` before running this test" unless defined? Primer::Docs
```

Comment:
I think it might be better to raise here instead of silently skipping in case something changes in  in the CI workflow where previews isn't generated.